src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -345,6 +346,10 @@ public function load(array $configs, ContainerBuilder $container)
             }
         }
 
+        if (!class_exists(AbstractUid::class)) {
+            $container->removeDefinition('console.command.uid_generate');
```

Comment:
I'd suggest moving this next to line 188 where there are similar checks

Reply:
Done.